.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "2.7"
+
+# command to install dependencies
+install:
+  - "pip install ."
```

Comment:
ah okay, i'll assume that past-me knew something that present-me has forgotten. :-P looks fine to me.